code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -102,7 +102,8 @@
 		AC.Remove(src)
 	Target = null
 	Leader = null
-	Friends.Cut()
+	for(var/friend in Friends)
+		clear_friend(friend)
```

Comment:
IIRC this shouldn't be required due to `/datum/proc/Destroy` already clearing out all registered signals. Both on the datum itself and the registrations the datum has on others

Reply:
removed